#330 Design: 게시판 디자인 변경

JIN921 opened the pull request:
## 1. 무슨 이유로 코드를 변경했나요?
게시판의 변경된 UI를 적용했습니다! 

## 2. 어떤 위험이나 장애를 발견했나요?
글쓰기 버튼을 fixed로 하단부에 고정 시켰는데
![image](https://github.com/user-attachments/assets/4cf898b8-92ef-4ed6-9e21-d42acb9b0732)
이렇게 보이다가 스크롤을 내려야 제대로 보입니다,,, 원인 아는 사람...
![image](https://github.com/user-attachments/assets/535c8026-b87a-43c1-b52c-ad9c2f2f9cd2)

## 3. 완료 사항
- [x] 자유게시판
- [x] 상단부 공지사항 컴포넌트
- [x] 자유게시판 탭
- [x] 게시물 목록 아이템 디자인 수정
- [x] 글쓰기 버튼 추가
- [x] 최신 공지사항 api 함수 추가

- 포지션 아이콘
- [x] 게시판 목록
- [x] 게시물 상세
- [x] 댓글
- [x] 대댓글

## 4. 추가 사항
공지사항 글쓰기 버튼은 어드민 처리하면서 같이 하려고 합니다~!!

yezzan9 commented:
> 이렇게 보이다가 스크롤을 내려야 제대로 보입니다,,, 원인 아는 사람...

```
position: fixed;
bottom: 아래 여백
```
이렇게 해보셨나용?!

yezzan9 commented on src/hooks/setPositionIcon.tsx:
이거 멤버 페이지에서도 활용할 수 있겠네요👀

yezzan9 commented on src/pages/Notice.tsx:
```diff
@@ -39,19 +38,21 @@ interface Content {
-const NoticeBoard = () => {
+const Notice = () => {
   const navigate = useNavigate();
   // TODO: 어드민인지 확인해서 true false 변경해주기
-  const isPostButtonVisible = true;
+  // const isPostButtonVisible = true;
```
해당 값은 왜 주석처리가 된 건가용?! 공지사항을 작성할 수 있는 글쓰기 버튼이 없는 것 같아요!!

JIN921 replied:
이건 이후 이슈에서 처리하려고 일단 주석처리 해놨습니다!

JIN921 commented:
> > 이렇게 보이다가 스크롤을 내려야 제대로 보입니다,,, 원인 아는 사람...
> 
> ```
> position: fixed;
> bottom: 아래 여백
> ```
> 
> 이렇게 해보셨나용?!

```
  position: fixed;
  bottom: 10px;
```
넵 이렇게 설정 해놧어요,,
근데 다시 하니까.. 되네요..!? 알 수 없다...
